unattended.sh

```diff
@@ -26,18 +26,22 @@ wizard_autoplay=${ENV_WIZARD_AUTOPLAY:-"false"} #Default the install wizard is n
 central_ip=${ENV_CENTRAL_IP:-$default_ip}       #Default central ip is the first of hostname -I
 
 function genpasswd() {
-	local _pwd
-	_pwd=$(date +%s%N | sha256sum | base64 | head -c 32)
+        local _pwd
```

Comment:
we should always keep the same indentation
```suggestion
  local _pwd
```